core/src/test/java/org/elasticsearch/search/query/QueryStringIT.java

```diff
@@ -223,11 +217,11 @@ public void testExplicitAllFieldsRequested() throws Exception {
         indexRandom(true, false, reqs);
 
         SearchResponse resp = client().prepareSearch("test2").setQuery(
-                queryStringQuery("foo eggplent").defaultOperator(Operator.AND)).get();
+                queryStringQuery("foo eggplant").defaultOperator(Operator.AND)).get();
         assertHitCount(resp, 0L);
 
         resp = client().prepareSearch("test2").setQuery(
-                queryStringQuery("foo eggplent").defaultOperator(Operator.AND).useAllFields(true)).get();
+                queryStringQuery("foo eggplant").defaultOperator(Operator.AND).useAllFields(true)).get();
```

Comment:
The `useAllFields(...)` is no tested by changing the default search field to `f1` instead of using a custom analyzer with ngram token filters.